cmd/edge-proxy/main.go

```diff
@@ -79,6 +82,17 @@ func main() {
 		os.Exit(1)
 	}
 
+	if externalHTTPProxyURI != "" {
+		parsedExternalProxy, err := url.Parse(externalHTTPProxyURI)
```

Comment:
We usually handle the unused variable the following:
```
_, err := url.Parse(externalHTTPProxyURI)
```

Reply:
I'm new to go.  I thought I tried that and it didn't like the leading _ .  I'll try again.  That's definitely cleaner